userspace/falco/app_actions/load_plugins.cpp

```diff
@@ -27,121 +27,34 @@ application::run_result application::load_plugins()
 		return run_result::fatal("Can not load/use plugins with musl optimized build");
 	}
 #endif
+	auto empty_src = state::source_info{};
 
-	// By default only the syscall event source is loaded and enabled
-	m_state->loaded_sources = {falco_common::syscall_source};
-	m_state->enabled_sources = {falco_common::syscall_source};
+	// Initialize the set of loaded event sources. 
+	// By default, the set includes the 'syscall' event source
+	m_state->sources.clear();
+	m_state->sources.insert(empty_src, falco_common::syscall_source);
+	m_state->loaded_sources = { falco_common::syscall_source };
```

Comment:
Oh, we always load `syscall_source`, i see.
Is there any reason behind that?

Reply:
See comment above